UK TL: A lot of stuff

tsunamistate opened the pull request:
Wanted to only translate the spell book but went on a roll
@SourceCodeDeleted please take a look
Because of the chaotic TL process, I just squashed everything into one commit, which might make it hard to review. Just ignore the lines where only numbers changed. Split view might make it easier
![image](https://user-images.githubusercontent.com/4958520/143678835-00bf1805-e9bb-4b1b-8847-91714196f332.png)

TL'd sections:
* Spellbook with new UI
* Settings menu
* Shrine names
* Towner names
* Shop UI
* Container names
* Item descriptions
* Book titles
* Character screen
* Quest names
* Spells
* Up to (level) - Down to (level) text

Might've missed something. Like I said, I went on a roll 😊

AJenbo commented on Translations/uk.po:
```diff
@@ -9413,123 +9625,129 @@ msgstr ""
 msgid "Back to Level {:d}"
-msgstr ""
+msgstr "Назад на {:d} рівень"
+
+#~ msgid "Replay Intro"
+#~ msgstr "Зіграти Заставку"
+
+#~ msgid "Extras"
+#~ msgstr "Додатки"
```
```suggestion
```

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -1573,122 +1522,122 @@ msgstr "Швидкість: Швидка"
-#: Source/help.cpp:38
+#: Source/help.cpp:39
 msgid "Z: Zoom Game Screen"
 msgstr "Z: Збільшити/зменшити гру"
```
Maybe is better екран?

tsunamistate replied:
I'll use "екран гри"

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -3587,731 +3542,724 @@ msgstr ""
 msgid "Gladiator's Ring"
 msgstr ""
```
Maybe?
Перстень гладіатора ?

SourceCodeDeleted replied:
Sorry maybe this is not needed yet.

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -5414,554 +5362,761 @@ msgstr ""
+#: Source/objects.cpp:118
 msgid "Eerie"
-msgstr ""
+msgstr "Мотрошний"
```
maybe Мотoрошний

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -5414,554 +5362,761 @@ msgstr ""
+#: Source/objects.cpp:129
 msgid "Glimmering"
-msgstr ""
+msgstr "Блмаючий"
```
maybe Блимаючий

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -5984,785 +6139,842 @@ msgstr "карта"
 #: Source/stores.cpp:319
 msgid "Buy premium items"
-msgstr ""
+msgstr "Купити преміальні предметі"
```
maybe предмети"

SourceCodeDeleted commented on Translations/uk.po:
```diff
@@ -5984,785 +6139,842 @@ msgstr "карта"
 #: Source/stores.cpp:320 Source/stores.cpp:673
 msgid "Sell items"
-msgstr ""
+msgstr "Продати предпети"
```
предмети

SourceCodeDeleted commented:
ok I looked it over. Most of it is really good. Some spelling mistakes. 

I think I can see when you became tired.

tsunamistate commented:
@SourceCodeDeleted and that's why we need reviews :)
Fixed all mentioned + found a new one where I missed a space in "Journal: The Ensorcellment"
+ added TL for "The Wages of Sin are War" and "The Binding of the Three"

SourceCodeDeleted reviewed (dismissed):
Everything looks good here.

tsunamistate commented:
@AJenbo good to merge

tsunamistate commented:
Rebased and squashed all commits into one
